Source/WebKit/Platform/cocoa/CocoaHelpers.mm

```diff
@@ -440,6 +444,40 @@ void callAfterRandomDelay(Function<void()>&& completionHandler)
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, delay.nanosecondsAs<int64_t>()), dispatch_get_main_queue(), makeBlockPtr(WTFMove(completionHandler)).get());
 }
 
+NSSet *availableScreenScales()
+{
+    NSMutableSet *screenScales = [NSMutableSet set];
```

Comment:
Do we use any generics in WebKit?

Reply:
No, because they conflict with C++ templates in C++ headers. We can use them in the mm files, but its complicated when the header needs to declare the method.